### ISSUE TITLE
--allow-const-label doesn't help

### ISSUE THREAD

**Issue by stiv-yakovenko** [catboost/catboost#2360]
Problem:
catboost version: 1.1.1
Operating System: Windows
CPU:

Dataset file 20230422_001424.csv

```
f1,f2,f3,f4,f5,f6,f7,f8,f9,f10,f11,f12,f13,f14
0,0,0,0,1,0,0,0,0,1,0,0,3,0,1,1306.0
0,0,0,0,1,0,0,0,0,1,0,0,3,0,0,9.0
```

Aux file: 20230422_001424.csv.cd

```
14	Label
15	Weight
```

Running command:
```
catboost-1.1.1.exe fit --allow-const-label --model-format CatboostBinary,json --cd 20230422_001424.csv.cd --has-header  --task-type CPU -f 20230422_001424.csv --train-dir catboost.train.dir --delimiter "," --verbose 100 -i 100 --loss-function Logloss --model-file catbm
```

I get exception: 

/catboost/libs/data/quantization.cpp:2416: All features are either constant or ignored

Adding flag --allow-const-label  doesn't change nothing.

**Comment by andrey-khropov** [comment]
And what do you expect? In this example all features are indeed constant, ML algorithms can't work on datasets with constant features.

**Comment by stiv-yakovenko** [comment]
No, ML algorithms can work on datasets with constant features, in this case model should output 0.9931. If I add string like this to the bottom of the file:
0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
I indeed get the correct answer, but it should happen automatically with --allow-const-label

**Comment by andrey-khropov** [comment]
> No, ML algorithms can work on datasets with constant features, in this case model should output 0.9931. If I add string like this to the bottom of the file: 0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0 I indeed get the correct answer, but it should happen automatically with --allow-const-label

--allow-const-label is not related to this error. It allows all samples of the dataset to have the same 'Label' value: https://catboost.ai/en/docs/references/training-parameters/#allow_const_label.

The problem in your dataset is with constant features, not constant label.

**Comment by andrey-khropov** [comment]
Duplicate of #1838
